Prevent scrolling when hamburger menu opened

### Discussion

Currently when the hamburger menu is opened, the user could still scroll the page.
Usually when a fullscreen overlay appears, the page behind would be "locked" for a better user experience.

I would love to create a PR to adjust the behavior 😊

### Validations

- [X] Follow our [Code of Conduct](https://github.com/carloscuesta/gitmoji/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/carloscuesta/gitmoji/blob/master/.github/CONTRIBUTING.md).
- [X] Check that there isn't already an issue requesting the same feature.

Hi @carloscuesta,

I followed the instructions from [How to start the website](https://github.com/carloscuesta/gitmoji/issues/1629#how-to-start-the-website) in the contributing guidelines for setting up the environment. However, I encountered an error while trying to commit my changes locally. The error message I received is:

```
$ git commit -m "🚧 WIP"
✔ Preparing lint-staged...
⚠ Running tasks for staged files...
  ❯ packages/website/.lintstagedrc.json — 2 files
    ❯ ./src/**/*.{ts,tsx,css} — 2 files
      ✖ eslint --cache --fix [FAILED]
      ◼ prettier --write ./src/**/*.{ts,tsx,css}
↓ Skipped because of errors from tasks.
✔ Reverting to original state because of errors...
✔ Cleaning up temporary files...

✖ eslint --cache --fix:

/Users/████████/gitmoji/packages/website/src/utils/theme/theme.css
  1:0  error  Parsing error: Declaration or statement expected

✖ 1 problem (1 error, 0 warnings)
```

If I only change .tsx files and commit, it will be fine.
But once change any CSS file, even if I add a space or a blank line and commit, it throws the error.
I'm on macOS Sonoma 14.1, using Git with Terminal.
Not quite sure what I was doing wrong here 🤔

I would appreciate your guidance on resolving this issue.
If you need any information please let me know. Thank you 🙏

--- a/packages/website/src/components/Layout/Hamburger/index.tsx
+++ b/packages/website/src/components/Layout/Hamburger/index.tsx
@@ -20,6 +20,14 @@ const Hamburger = () => {
     return () => Router.events.off('routeChangeStart', onRouteChangeStart)
   }, [])
 
+  useEffect(() => {
+    if (isOpen) {
+      document.body.classList.add('overflow-hidden')
+    } else {
+      document.body.classList.remove('overflow-hidden')
+    }
+  }, [isOpen])
+
   return (
     <div className={styles.hamburger}>
       <button

--- a/packages/website/src/utils/theme/theme.css
+++ b/packages/website/src/utils/theme/theme.css
@@ -114,6 +114,10 @@ pre {
   padding: 1em;
 }
 
+.overflow-hidden {
+  overflow: hidden;
+}
+
 .overflow-x-adjust {
   overflow-x: auto;
 }

